eva-accession-release-automation/publish_release_to_ftp/README_release_changelog.txt

```diff
@@ -0,0 +1,34 @@
+This document describe technical changes associated with the EVA RefSNP release.
+
+EVA RefSNP release 5
+====================
+
+- Release is now performed per taxonomy of the submitted variants:
+  Previous release contain variant based on their presence on an assembly.
+  From release 5, a variant will only be present if it was found in the specified taxonomy.
+- The variants previously marked as multi-mapped have now been deprecated and are available in the deprecated file
```

Comment:
Suggest summarizing both these categories (multimap and alleles mismatch) as deprecated instead of two separate sentences...

Reply:
These are coming from two different sources (release under multi-mapped or not released at all), so I think I prefer to keep them separated.